### PR TITLE
added breif set up guide

### PR DESCRIPTION
**Pull request by mdewey**
## Description
As I was ramping up, I found a gap in the documentation around setting up local authentication. If this already exists, feel free to close the issue with a link to the existing docs. 

## Testing done
Followed the documentation locally to verify that the steps are inclusive. 

## Screenshots
- N/A

## Acceptance criteria
- [ ] The documentation is completed

## Definition of done
- [X] Events are logged appropriately (No events)
- [X] Documentation has been updated, if applicable
- [X] No link to an issue.
- [X ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: docs/HowToLogInLocally.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## To log in:
+0) Have both the website and API running. 
+1) Head to [flipper for the API](http://localhost:3000/flipper/features)
+2) Disable the `ssoe` feature flag. 
+3) Go your local website, and select sign in and sign in with `ID.me`
+4) Log in with a appropriate user, ask your team for what can be used. 
+5) Click through the next few pages, it won't send a text, and it should allow you to pass right on through. 
+6) You should be redirected back to the home page, signed in.
+7) To see if it worked, you should be able to open your [redux dev tools](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd) and see the user object in the global state.
```
Comment (ncksllvn) [inline docs/HowToLogInLocally.md:19]: Each item in this list can just start with `1.`  and GitHub MD will automatically enumerate it

**Review by ncksllvn — APPROVED** [review]
LGTM, I think this would be a useful guide for a FE engineer going through onboarding

**Comment by rianfowler** [comment]
@department-of-veterans-affairs/vsp-identity LMK if these instructions are accurate and if this is where we want to document this info.

File: docs/HowToLogInLocally.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+## Prerequisites
+
+- Have the [site](https://github.com/department-of-veterans-affairs/vets-website) building locally, and the drupal content updated. 
+- Have the [API](https://github.com/department-of-veterans-affairs/vets-api) running locally, with [mock-api-data](https://github.com/department-of-veterans-affairs/vets-api-mockdata), either in docker or locally
+
+
+## To log in:
+0) Have both the website and API running. 
+1) Head to [flipper for the API](http://localhost:3000/flipper/features)
+2) Disable the `ssoe` feature flag.
```
Comment (jholton) [inline docs/HowToLogInLocally.md:14]: Can we remove these lines?
SSOe should be working locally now, and it'll be the only option in the near future (e.g. the `ssoe` feature flags will be removed and everyone will use SSOe login).
If you're having trouble with SSOe login, let me know!

Comment (rianfowler) [reply docs/HowToLogInLocally.md:14]:
```suggestion
```

**Review by rianfowler — APPROVED** [review]
update the SSOE instructions befrore merging

**Pull request closed without merging**